README.md

```diff
@@ -333,6 +333,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [rTorrent](http://libtorrent.rakshasa.no/)
 - [R](http://www.r-project.org/)
 - [Redshift](http://jonls.dk/redshift/)
+- [Rime](rime.im)
```

Comment:
not a real url, fixed in https://github.com/lra/mackup/commit/a9dae3d799783ccca33a30879c50008415678103